p2p_wallet/Scenes/Main/SendToken/Subscenes/ChooseRecipientAndNetwork/SelectAddress/SendToken.ChooseRecipientAndNetwork.SelectAddress.ViewModel.swift

```diff
@@ -23,6 +23,7 @@ protocol SendTokenChooseRecipientAndNetworkSelectAddressViewModelType: WalletDid
     var networkDriver: Driver<SendToken.Network> { get }
     var payingWalletDriver: Driver<Wallet?> { get }
     var feeInfoDriver: Driver<Loadable<SendToken.FeeInfo>> { get }
+    var warning: Driver<String> { get }
```

Comment:
the name warning is not clearly define the type of variable, should be warningDriver